src/pages/CreateSalary/CreateSalary.tsx

```diff
@@ -1,120 +1,270 @@
+                    companySize: e.detail.value! as string,
+                  })
+                }
+                className="input ion-margin-top"
+                value={formNewSalary.companySize}
+                type="text"
+              ></IonInput>
+            </div>
+            <div className="input-wrapper">
+              <IonLabel class="label">Gross Annual Salary (€)</IonLabel>
+              <IonInput
+                onIonChange={(e) =>
+                  setFormNewSalary({
+                    ...formNewSalary,
+                    grossSalary: parseInt(e.detail.value!, 10),
+                  })
+                }
+                type="number"
+                className="input ion-margin-top"
+                value={formNewSalary.grossSalary}
+              ></IonInput>
+            </div>
+
+            <div className="input-wrapper">
+              <IonLabel class="label">Location</IonLabel>
+              <IonInput
+                onIonChange={(e) =>
+                  setFormNewSalary({
+                    ...formNewSalary,
+                    location: e.detail.value! as string,
+                  })
+                }
+                className="input ion-margin-top"
+                value={formNewSalary.location}
+                type="text"
+              ></IonInput>
+            </div>
+
+            <div className="input-wrapper">
+              <IonLabel class="label">Gender</IonLabel>
+              <IonInput
+                onIonChange={(e) =>
+                  setFormNewSalary({
+                    ...formNewSalary,
+                    gender: e.detail.value! as string,
+                  })
+                }
+                className="input ion-margin-top"
+                value={formNewSalary.gender}
+                type="text"
+              ></IonInput>
+            </div>
+
+            <div className="input-wrapper">
+              <IonLabel class="label">Is your position fully remote? </IonLabel>
+              <IonToggle
+                checked={formNewSalary.fullRemote}
+                class="ion-margin-top ion-justify-content-start"
+                onIonChange={(e) =>
+                  setFormNewSalary({
+                    ...formNewSalary,
+                    fullRemote: e.detail.checked,
+                  })
+                }
+              />
+            </div>
+
+            <div className="input-wrapper">
+              <IonLabel class="label">Weekly Hours</IonLabel>
+              <IonInput
+                onIonChange={(e) =>
+                  setFormNewSalary({
+                    ...formNewSalary,
+                    weeklyHours: parseInt(e.detail.value!, 10),
+                  })
+                }
+                type="number"
+                className="input ion-margin-top"
+                value={formNewSalary.weeklyHours}
+              ></IonInput>
+            </div>
+            <div className="input-wrapper ion-align-self-start">
```

Comment:
What's the expected behaviour of this field? I'd expect that when I add a new technology, it is added to the textbox, but that's not the case

Reply:
they should be added to a list that will appear on the top of the input:
![image](https://user-images.githubusercontent.com/4491909/79116208-1f289900-7d88-11ea-9189-6010b219b091.png)

Reply:
I don't understand how I didn't see it yesterday... Sorry!